gatsby-config.js

```diff
@@ -30,6 +31,23 @@ module.exports = {
         `https://widget.trustpilot.com/bootstrap/v5/tp.widget.bootstrap.min.js`,
     ],
     plugins: [
+        {
+            resolve: 'gatsby-plugin-sass',
```

Comment:
Dose this plugin works with Gatsby 5? :grin:

Reply:
yes it will.